docs/UserGuide.md

```diff
@@ -183,6 +191,38 @@ Expected output (fail):
 ```agsl
 Oops! Invalid fosterer index provided, please check again.
 ```
+### Saving changes in a fosterer's details: `save`
+
+Saves changes in details of the fosterer made in the profile page. 
+<div markdown="span" class="alert alert-primary">
+  :exclamation: <b>Important:</b>
+This command is only available while in fosterer profile view page. 
+This command works only in a profile view page, which  after executing a view command.
+</div> 
+
+Format: `save`
+
+Expected Output (if successful): 
+```agsl
+Edited Fosterer: [NAME]; Phone: [NUMBER]; Email: [EMAIL]; Address: [ADDRESS]; Housing: [HOUSING]; Availability: [AVAILABILITY]; Animal name: [ANIMAL NAME]; Animal type: [ANIMAL TYPE]; Tags: [TAG]
+```
+
+
+Expected output (if successful):
+
+| Scenario                                                                                                                        | Error Message                                                                                                                                                                                                                                                                                                                                                                                                                                                        |
+|---------------------------------------------------------------------------------------------------------------------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
```

Comment:
looks okay to me now since the error messages are not confirmed for this feature yet...?